Sources/Alchemy/Database/QueryBuilder/Types/SQL.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public struct SQL {
-    let query: String
+    var query: String
```

Comment:
For modification of inserts @ the database level. (i.e. adding "RETURNING *" so we can get any database default values that are set)